core/src/main/scala/kafka/tools/PerfConfig.scala

```diff
@@ -39,6 +39,8 @@ class PerfConfig(args: Array[String]) {
     .defaultsTo("yyyy-MM-dd HH:mm:ss:SSS")
   val showDetailedStatsOpt = parser.accepts("show-detailed-stats", "If set, stats are reported for each reporting " +
     "interval as configured by reporting-interval")
+  val showAllStatsOpt = parser.accepts("show-all-stats", "Show intermediary stats based on the " +
```

Comment:
Do we really need a new option for this? I was a bit surprised that `showDetailedStats` doesn't show the final result to be honest. You probably did this for compatibility reasons, but it would be good to get a second opinion from @gwenshap or @junrao.